### DISCUSSION TITLE
Set TextInput Length

### DISCUSSION THREAD

**Discussion by he0xA1** [slint-ui/slint#2594] [Q&A]
how can I set the length for input or check it?

**Comment by ogoffart** [comment]
Hi, What do you mean by the length?  Is is the amount of character that the user can enter?

We currently don't have code in .slint itself to do that. But you can get some validation in native code.
One example that does that is the booker example.

https://github.com/slint-ui/slint/blob/master/examples/7guis/booker.rs
